Version préliminaire de l'IG préadmission en ligne

NicolasMoreauCPage opened the pull request:
## Description des changements

Première version de l'IG : 
* Ajout des profiles, extensions, valuesets et codesystems
* Ajout du narratif
* Ajout des exemples

## Preview

https://interop-sante.github.io/hl7.fhir.fr.preadmission/nmoreau/ig

NicolasMoreauCPage committed:
Saisie initiale

NicolasMoreauCPage committed:
configuration du sushi-config.yaml et du ig.ini

NicolasMoreauCPage committed:
Corrections diverses sur les consentements et les autres ressources, et refonte du narratif et ajout d'exemples

NicolasMoreauCPage committed:
Ajout couverture coiale + narratif + exemples

NicolasMoreauCPage committed:
correction narrative sur index.md

nriss committed:
Merge branch 'main' into nmoreau

nriss commented on input/fsh/Aliases.fsh:
```diff
@@ -0,0 +1,3 @@
+Alias: FrCorePatient = https://hl7.fr/ig/fhir/core/StructureDefinition/fr-core-patient
```
Je pense que ce n'est pas nécessaire, tu peux mettre directement FrCorePatient dans les profils, FrCorePatient sera directement reconnu comme il est en dépendance dans sushi-config

nriss commented on sushi-config.yaml:
```diff
@@ -20,20 +21,52 @@ jurisdiction: urn:iso:std:iso:3166#FR "FRANCE"
-#   ans.fr.nos: latest
+  
+dependencies:
+  hl7.fhir.fr.core: 2.0.1
```
```suggestion
  hl7.fhir.fr.core: 2.1.0
```

nriss commented on input/pagecontent/sushi-config.md:
```diff
@@ -0,0 +1,24 @@
+# sushi-config.md
```
A quoi correspond ce fichier ? Pourquoi s'appelle-t-il sushi-config ?

nriss commented on input/fsh/codesystems/PreadmissionCodeSystemDocumentTypeFr.fsh:
```diff
@@ -0,0 +1,22 @@
+* ^status = #active
+* ^content = #complete
+
+* #CNI "Carte d’identité"
```
Il est probablement possible de réutiliser ce CodeSystem ici : 
https://hl7.fr/ig/fhir/core/CodeSystem-fr-core-cs-mode-validation-identity.html

C'est les modes de validation de l'identité patient dans le cadre de l'INS

nriss commented on input/fsh/resources/appointment-irm.json:
```diff
@@ -0,0 +1,41 @@
+{
```
Je ne comprend pas trop l'arborescence, pourquoi as-tu un dossier instances et un autre resources ?
Est-il possible de tout mettre dans un dossier example au format fsh ?

nriss commented on input/fsh/valuesets/PreadmissionConsentCategoryValueSet.fsh:
```diff
@@ -0,0 +1,8 @@
+Id: consent-category-value-set
+Title: "ValueSet pour la catégorie de consentement"
+Description: "Valeurs autorisées pour la catégorie de consentement"
+* ^url = "http://hl7.fr/fhir/fr/preadmission/ValueSet/consent-category-value-set"
```
Tu peux supprimer l'url, la version et le status de l'ensemble des VS et CS. Ces attributs seront automatiquement complétés en accord avec ce qu'il est indiqué dans le sushi-config.

nriss commented on sushi-config.yaml:
```diff
@@ -1,7 +1,7 @@
 canonical: http://hl7.fr/fhir/fr/preadmission
-name: PREAD
-title: Préadmission en ligne
+name: PreadmissionFHIRIG
```
```suggestion
name: PREAD
```

nriss replied:
Je propose d'utiliser plutôt un code en majuscule pour le name

Selon la doc :
<img width="1206" alt="image" src="https://github.com/user-attachments/assets/94e00b62-a827-4ba6-9240-5a62c366ca57" />

nriss reviewed (changes requested):
Hello, j'ai fait une première passe rapide :)

Tout est bien sûr discutable!

NicolasMoreauCPage committed:
Suppression des URL en dur dans les fsh

NicolasMoreauCPage committed:
Corrections suite au retour de la PR

NicolasMoreauCPage committed:
Modification sur le narratif

NicolasMoreauCPage commented:
Merci pour tes retours. J'ai apporté les corrections.

NicolasMoreauCPage committed:
changement de l'url  canonical par https://github.com/Interop-Sante/hl7.fhir.fr.preadmission pour résoudre les problèmes d'URL remontés par le l'interpreteur d'InteropSanté

nriss commented on sushi-config.yaml:
```diff
@@ -1,7 +1,7 @@
-canonical: http://hl7.fr/fhir/fr/preadmission
-name: PREAD
-title: Préadmission en ligne
+canonical: https://github.com/Interop-Sante/hl7.fhir.fr.preadmission
```
Excuse moi pour t'avoir induit en erreur sur ce point, l'url précédente http://hl7.fr/fhir/fr/preadmission était bonne, cette erreur sera résolue automatiquement lors de la publication
Je pensais notamment aux profils ShareableValueSet et ShareableCodeSystem que tu peux utiliser

NicolasMoreauCPage replied:
OK merci, je revert l'url et je vais regarder ces profils

NicolasMoreauCPage committed:
Ajout des exemples manquants, restriction du appointement sur le FRpractionner au lieu de Practitionner et lien avec FrCorePatientProfile au lieu de FrCorePatientINSProfile

NicolasMoreauCPage committed:
revert sur le changement d'url

NicolasMoreauCPage committed:
ajout d'une page d'historique

NicolasMoreauCPage committed:
Correction des CS et VS pour la conformité avec ShareableCS/VS + ajout des diagrammes pour les annulations + ajout descriptif dans les exemples

nriss commented on input/fsh/codesystems/AlimentationDMPConsentCodeSystems.fsh:
```diff
@@ -0,0 +1,9 @@
+Description: "Valeurs possibles pour le consentement à l’alimentation du DMP."
+* ^content = #complete
+* ^experimental = true
+* #NA "Information non recueillie"
```
Il faudrait voir avec @ypoiron si l'on peut uniformiser certains CodeSystems avec PAM
Il faut également qu'on réfléchisse à l'utilisation de codes SNOMED plutôt que d'utiliser des codes franco-français

NicolasMoreauCPage replied:
J'avais repris les valeurs définies pour PAM, il y a vait 2 jeux de valeurs pour chacun des consentemens DMP mais portant les mêmes informations, je les ai unifié en utilsant qu'un seul de ces jeux. J'ai regardé coté SNOMED, je trouve les codes pour "consentements non recceuili" et "consentement éclairé accepté" mais pas pour le refus. La partie consentement est encore a travailler pour modéliser l'ensemble de ses consentements

nriss replied:
Top merci pour tes recherches @NicolasMoreauCPage !
Idéalement, on devrait créer un IG terminologies qui permet d'uniformiser toutes les terminos InteropSanté, un peu comme fait ici : https://terminology.hl7.org

Peut être à faire dans un second temps quand on aura gagné en maturité sur nos process côté IS

nriss commented on input/fsh/examples/appointment-irm.fsh:
```diff
@@ -0,0 +1,18 @@
+* reasonCode[0].coding[0].system = "http://snomed.info/sct"
+* reasonCode[0].coding[0].code = #241541005
+* reasonCode[0].coding[0].display = "IRM cérébrale (procédure)"
+* extension[0].url = "http://hl7.fr/fhir/fr/preadmission/StructureDefinition/appointment-context"
```
Tu peux dire le nom de l'extension quand tu crées des exemples pour que l'url soit remplie automatiquement
Je l'ai fait ici par exemple : https://github.com/ansforge/IG-fhir-mesures-de-sante/blob/1c46eb598fafcc1e09ac590e2f564ba1301b9a1b/input/fsh/Examples/Examples1.fsh#L61

nriss commented on input/fsh/examples/coverage-amc.fsh:
```diff
@@ -0,0 +1,33 @@
+* period.end = "2025-12-31T23:59:59+01:00"
+
+// Extension pour les informations spécifiques à l'AMC
+* extension[0].url = "http://fhir_interop.happytal.com/fhir/StructureDefinition/FrCoverageAMCExtended"
```
A quoi correspond cette URL ?

nriss commented on input/fsh/examples/coverage-amc.fsh:
```diff
@@ -0,0 +1,33 @@
+
+// Extension pour les informations spécifiques à l'AMC
+* extension[0].url = "http://fhir_interop.happytal.com/fhir/StructureDefinition/FrCoverageAMCExtended"
+* extension[0].extension[0].url = "nomAMC"
```
Idem, tu peux rajouter le nom des slices / extensions au lieu des index, ça permet de mieux voir de quoi on parle

nriss commented on input/fsh/examples/document-cni-recto.fsh:
```diff
@@ -0,0 +1,17 @@
+* type.coding[0].system = "http://hl7.fr/fhir/fr/preadmission/CodeSystem/document-type-code-system"
+* type.coding[0].code = #CNI
+* type.coding[0].display = "Carte Nationale d'Identité - Recto"
+* subject.reference = "Patient/patient-001"
```
Tu peux faire les refs comme ça : * subject = Reference(ExamplefrPatient001) "Pierre Durand"
https://github.com/ansforge/IG-fhir-mesures-de-sante/blob/1c46eb598fafcc1e09ac590e2f564ba1301b9a1b/input/fsh/Examples/Examples1.fsh#L58

nriss commented on sushi-config.yaml:
```diff
@@ -12,6 +12,7 @@ contact:
         value: fhir@interopsante.org
         use: work
 status: draft
+url : https://github.com/Interop-Sante/hl7.fhir.fr.preadmission
```
Je t'invite à enlever la ligne url et de laisser la ligne par défaut
Quand tu regardes la description, ça ne correspond pas au repo GitHub

<img width="938" alt="image" src="https://github.com/user-attachments/assets/6fba9c4f-c9be-4097-89da-d045f5831971" />

nriss commented on input/fsh/extensions/PreadmissionStatutFr.fsh:
```diff
@@ -0,0 +1,12 @@
+Extension: PreadmissionStatutFr
+Id: hl7.fhir.fr.preadmission.statut-preadmission
```
Rajouter title et description peut être ?

NicolasMoreauCPage committed:
Corrections demandées en PR + travail sur les consentements

nriss commented on input/markdown/profiles/StructureDefinition-index.md:
```diff
@@ -0,0 +1,8 @@
+# Profils FHIR personnalisés
```
Sais-tu comment accéder à cette page ? Je ne la trouve pas dans la preview

nriss commented on input/pagecontent/appointment.md:
````diff
@@ -0,0 +1,113 @@
+
+Voici un exemple minimal de ressource `Appointment` pour un rendez-vous d’IRM cérébrale :
+
+```json
````
Plutôt que de mettre directement le json dans le md, je t'invite plutôt à créer un exemple et l'inclure comme on l'a fait dans le cadre du SAS ici :

https://github.com/ansforge/IG-fhir-service-acces-aux-soins/blob/d4db1d2681cd9139e80d614398414e466d32dd97/input/pagecontent/specifications_techniques-tous-gestion_regulateur.md?plain=1#L102
https://ansforge.github.io/IG-fhir-service-acces-aux-soins/main/ig/specifications_techniques-sos-gestion_regulateur.html

<img width="1113" alt="image" src="https://github.com/user-attachments/assets/0358a633-3c64-45e2-a536-f44faf063950" />


Ca évitera d'aller chercher difficilement tous les exemples dans les pages dès que l'on fera des mises à jours de l'IG

nriss commented on input/pagecontent/appointment.md:
```diff
@@ -0,0 +1,113 @@
+# Rendez-vous dans la Preadmission
```
Pour éviter une numérotation foireuse, il faut commencer au niveau 3 (###) au sein des pages md et éviter de mettre un titre ici.
Si tu mets un titre ici, ça va faire doublon avec ce qui est dans le sushi config (title: Rendez-vous (Appointment))

<img width="1246" alt="image" src="https://github.com/user-attachments/assets/be58c228-d3f4-4e0d-9f27-9f468b0eb8ac" />

NicolasMoreauCPage committed:
Correctement sur les numérotations des pages + suppression de pages étant redondante avec la page des ressources de conformités générés automatiquement

nriss commented:
Hello @NicolasMoreauCPage :)

J'ai une nouvelle proposition à te faire concernant les pages md associées aux profils, comme celle-ci sur [Coverage](https://interop-sante.github.io/hl7.fhir.fr.preadmission/nmoreau/ig/coverage.html) !
C'est une proposition, n'hésite pas à me donner ton avis :)


Si tu utilises ce format de nom de page .md :
https://github.com/Interop-Sante/hl7.fhir.fr.core/blob/main/input/pagecontent/StructureDefinition-fr-core-patient-ins-intro.md
Elle va être mis directement dans l'encart intro du profil.
Exemple ici : https://hl7.fr/ig/fhir/core/StructureDefinition-fr-core-patient-ins.html

Si tu fais ça par exemple pour Coverage, ça permettrait d'avoir dans la même page la description narrative du profil avec la description technique (snapshot, differential, et tout ce qui va avec...)
Qu'en penses-tu ?

NicolasMoreauCPage committed:
Refonte des consentements. Le choix a été fait de définir les types de consentemetns dans les provisions et d'utiliser seulement des codes déjà définis. Le fait que les consentements n'ont pas été recceuilli est explicité par un non refus du consentements

NicolasMoreauCPage committed:
Déplacement du narratif sur les ressources dans les pages de conformité en tant qu'introduction

NicolasMoreauCPage commented:
@nriss oui bonne idée. J'ai les ai déplacé comme tu as indiqué.

J'ai refait toute la partie sur les consentements en n'utilisant que des codes existants. @ypoiron @nriss je vous laisse me faire vos retours.

nriss commented on input/pagecontent/StructureDefinition-preadmission-encounter-fr-intro.md:
```diff
@@ -0,0 +1,109 @@
+
+Le **statut de la préadmission** est représenté à l’aide de l’extension `PreadmissionStatutFr`. Cette extension permet de suivre l’état administratif de la préadmission avec les valeurs suivantes :
+
+- `CREATED` : Pré-admission créée mais non encore traitée.
```
Je te conseille de mettre le lien vers le ValueSet plutôt que de ré-énumérer l'ensemble des codes permis, ça peut engendrer des problèmes de maintenance plus tard parce qu'il faut penser à modifier à plusieurs endroits

nriss commented on input/pagecontent/StructureDefinition-preadmission-documentreference-fr-intro.md:
````diff
@@ -0,0 +1,93 @@
+
+Voici un exemple minimal de ressource `DocumentReference` pour une carte d’identité téléversée par le patient :
+
+```json
````
Je te conseille de créer une instance de DocumentReference et d'utiliser ce type de code pour afficher automatiquement le json de l'instance :

{%include Appointment-2-json-html.xhtml%}

nriss commented on input/pagecontent/StructureDefinition-preadmission-documentreference-fr-intro.md:
```diff
@@ -0,0 +1,93 @@
+
+Voici une liste des documents les plus couramment demandés lors de la préadmission, basée sur le `ValueSet` des types de documents autorisés :
+
+- 📄 **Carte nationale d’identité (CNI)** : Document officiel permettant d'attester de l'identité du patient.
```
Idem, il vaut mieux référencer le ValueSet plutôt que de répéter les codes utilisés.

nriss commented on input/fsh/profiles/PreadmissionEncounterFr.fsh:
```diff
@@ -0,0 +1,40 @@
+Title: "Fr Preadmission Encounter Profile"
+Description: "Profil FHIR pour un encounter lié à une préadmission"
+
+* extension contains EncounterPatientComment named remarquePatient 0..1
```
Peux-tu rajouter des ^short à chaque extension pour expliciter à quoi correspond chacune ?

nriss reviewed (commented):
Bravo pour ton travail!

nriss commented on input/fsh/codesystems/PreadmissionCodeSystemDocumentTypeFr.fsh:
```diff
@@ -0,0 +1,16 @@
+CodeSystem: PreadmissionDocumentTypeCodeSystem
```
Peux-tu rajouter le profil shareableCodeSystem et shareableValueSet ?

* ^meta.profile = "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
https://github.com/Interop-Sante/hl7.fhir.fr.core/blob/main/input/fsh/valuesets/FRCoreValueSetBPMeasurementMethod.fsh

nriss commented on input/fsh/extensions/PreadmissionEncounterAgentInstructions.fsh:
```diff
@@ -0,0 +1,8 @@
+Extension: EncounterAgentInstructions
+Id: encounter-agent-instructions
+Title: "Consignes de l'agent administratif"
+* ^description = "Consignes à destination du patient, fournies par l'agent administratif"
```
```suggestion
Description: "Consignes à destination du patient, fournies par l'agent administratif"

```

nriss commented on input/fsh/extensions/PreadmissionEncounterPatientComment.fsh:
```diff
@@ -0,0 +1,8 @@
+Extension: EncounterPatientComment
+Id: encounter-patient-comment
+Title: "Remarque du patient"
+* ^description = "Remarque libre saisie par le patient à destination de l’agent administratif"
```
```suggestion
Description: "Remarque libre saisie par le patient à destination de l’agent administratif"

```

nriss commented on input/pagecontent/StructureDefinition-appointment-questionnaire-response-intro.md:
```diff
@@ -0,0 +1,97 @@
+| `allergies`              | Souffrez-vous d’allergies connues ?                      | boolean  |
+| `allergies_details`      | Si oui, lesquelles ?                                     | string   |
+
+Exemple de structure FHIR (simplifiée) :
```
Cf. https://build.fhir.org/ig/FHIR/ig-guidance/fragments.html

nriss commented on input/fsh/examples/coverage-amc.fsh:
```diff
@@ -0,0 +1,33 @@
+* period.end = "2025-12-31T23:59:59+01:00"
+
+// Extension pour les informations spécifiques à l'AMC
+* extension[informationsAmc].url = "http://hl7.fr/fhir/fr/preadmission/StructureDefinition/fr-coverage-amc-extended"
```
Tu peux supprimer les champs fixés par l'extension tel qu'URL, ils seront complétés automatiquement

nriss commented on input/fsh/profiles/PreadmissionCoverageFr.fsh:
```diff
@@ -0,0 +1,54 @@
+* status = #active
+
+// Précision pour la date de début de couverture
+* period.start 1..1 MS
```
Après discussion avec Florian, il est inutile de rendre obligatoire la période.
Cf https://github.com/Interop-Sante/hl7.fhir.fr.preadmission/issues/5

nriss commented on input/fsh/profiles/PreadmissionCoverageFr.fsh:
```diff
@@ -0,0 +1,54 @@
+
+// Précision pour la date de début de couverture
+* period.start 1..1 MS
+* period.start only dateTime
```
A quoi sert cette ligne ?

nriss commented on input/fsh/profiles/PreadmissionCoverageFr.fsh:
```diff
@@ -0,0 +1,54 @@
+
+// Précision pour le champ subscriber
+* subscriber 1..1 MS
+* subscriber only Reference(FRCorePatientProfile)
```
L'assuré n'est pas forcément le patient, ça peut être le parent. Il faut dans ce cas utiliser le profil RelatedPerson

NicolasMoreauCPage committed:
Travail sur les consentements
+ corrections liés aux issues
+ revue du narratif
+ ajout d'un critère de recherche sur le statut de la préadmission pour le profil Encounter
+ ajout d'un capabilitystatement pour préciser les actions possibles sur les ressources

NicolasMoreauCPage committed:
Précisions sur les critère de recherche sur Coverage, DocumentReference et Encounter + nettoyage des fsh inutiles + précisions dans le narratif

nriss commented on input/resources/SearchParameter-encounter-preadmission-status.json:
Tu ne l'as pas faite en FSH ?

NicolasMoreauCPage replied:
J'avais essayé au départ mais j'avais des erreurs sushi m'indiquant qu'il ne connaissait pas SearParameter, je l'avais donc fait en JSON. Mais, suite à ton commentaire, j'ai pu le refaire sans erreur en fsh... surement une erreur de ma part à mon premier essai

nriss reviewed (approved):
Super boulot :)

NicolasMoreauCPage committed:
Reecriture du parametre de recherche en FSH

NicolasMoreauCPage committed:
Nettoyage des commentaires

NicolasMoreauCPage committed:
Ajout de la référence aux textes qui définissent la récote du consentement DMP - solve #9

NicolasMoreauCPage committed:
Changement en version 0.2.0

NicolasMoreauCPage committed:
Suppression de la limite sur Encounter.location - fix #14

NicolasMoreauCPage committed:
Suppressiojn de l'extension pour accéder aux consentements + description de l'accès - fix #10

NicolasMoreauCPage committed:
#10 - maj des exemples

NicolasMoreauCPage committed:
uppression d'une extension permettant de faire la référence Encounter --> QuestionnaireResponse - fix #11

NicolasMoreauCPage committed:
Correction des exemples + explication comment accéder aux consentements et aux réponses des questionnaie - fix #10 fix #11

NicolasMoreauCPage committed:
Suppression du paramètre nos dans le fhir-workflow.yml car il n'est pas reconnu par github : run-gitHubPages
Unexpected input(s) 'nos', valid inputs are ['ig-publisher-version', 'github_page_token', 'github_page', 'generate_plantuml', 'generate_mapping_plantuml', 'generate_testscript', 'repo_ig', 'bake', 'validator_cli', 'publish_repo', 'publish_repo_token', 'publish_path_outpout']

NicolasMoreauCPage committed:
Ajout d'une version fhir dans . ig.ini suite à une erreur de compilation "org.hl7.fhir.exceptions.FHIRException: Invalid candidate version: null" dans github

NicolasMoreauCPage committed:
Changement des classes d'admission #6 + Ajout des types de documents pour les personnes à prévenir et de confiance #16

NicolasMoreauCPage committed:
Mise a jour des exemples et du laius #6

nriss commented on input/fsh/examples/coverage-amc.fsh:
```diff
@@ -0,0 +1,32 @@
+* period.end = "2025-12-31T23:59:59+01:00"
+
+// Extension pour les informations spécifiques à l'AMC
+* extension[informationsAmc].extension[nomAMC].url = "nomAMC"
```
Je pense qu'il est préférable de ne pas écrire l'URL ici : elle est rajoutée automatiquement par sushi (il trouve l'url automatiquement à partir du nom de la slice).

Le remplissage automatique permet d'éviter de l'éditer à deux endroits en cas de modification

nriss commented on input/fsh/examples/coverage-amc.fsh:
```diff
@@ -0,0 +1,32 @@
+
+// Ajout d'un identifier
+* identifier[0].system = "http://example.com/identifiers/coverage"
+* identifier[0].value = "AMC-2025-001"
```
Une bonne pratique (mais c'est du détail) est d'utiliser les incréments plutôt que les chiffres

* identifier[0]
* identifier[=]
* identifier[=]

* identifier[+]
* identifier[=]
* identifier[=]

Cela permet de rajouter des éléments en plein milieu sans avoir à décaler tous les chiffres

nriss commented on input/fsh/examples/encounter-complete-par-patient.fsh:
```diff
@@ -0,0 +1,23 @@
+* period.start = "2025-05-15T10:30:00+01:00"
+* period.end = "2025-05-15T11:00:00+01:00"
+* identifier.use = #temp
+* identifier.type.coding.system = "http://interopsante.org/fhir/CodeSystem/fr-core-identifier-type"
```
Il serait préférable d'indiquer la slice d'identifier ici
(Idem pour l'ensemble des champs multivalués slicés)

nriss commented on sushi-config.yaml:
```diff
@@ -12,28 +12,45 @@ contact:
         use: work
 status: draft
-version: 0.1.0
+version: 0.2.0
```
Je pense que ça n'est pas nécessaire de changer le numéro de version tant que c'est pas publié !
Je propose de faire une première release avec le numéro de version "0.1.0-ballot" pour la mise en concertation

nriss commented on sushi-config.yaml:
```diff
@@ -12,28 +12,45 @@ contact:
-#   ans.fr.nos: latest
+  
+dependencies:
+  hl7.fhir.fr.core: 2.1.0
```
Je sais pas s'il y a un intérêt, mais il y a possibilité de rajouter ans.fr.terminologies en dépendance s'il y a utilisation des terminos du NOS / du SMT

nriss commented on ig.ini:
```diff
@@ -1,50 +1,50 @@
+# see comments below for instructions
+
+ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.fr.preadmission.json
+template = https://github.com/Interop-Sante/hl7.ig.fr.template
```
```suggestion
template = https://github.com/Interop-Sante/hl7.ig.fr.template/tree/template-IS-colors
```

Pour tester le nouveau template InteropSanté si ça te tente!

NicolasMoreauCPage committed:
MAJ fhir-workflows.yml

NicolasMoreauCPage committed:
Merge pull request #8 from Interop-Sante/nr-add-issue-link

Ajout du lien vers l'issue dans le footer de l'IG

NicolasMoreauCPage committed:
#13 - ajout de FrCorePatientINSProfile à Encounter et Appointment dans le cas de la répcetion d'une identité qualifié via AppCV

NicolasMoreauCPage committed:
fix #16 - maj des types de documents

NicolasMoreauCPage committed:
Correction des exemples

NicolasMoreauCPage committed:
#15 - déplacement du numero AMC en tant que identifier de Organization + MAJ des exemples + MAJ du narratif

NicolasMoreauCPage committed:
Ajout du descriptif de l'OID pour les nuero AMC

NicolasMoreauCPage committed:
Application du nouveau thèmes et corrections QA

Commit:
fix #17 : Suppression extension permettant de faire la référence Appointment--> QuestionnaireResponse

Commit:
#16 : ajout de https://smt.esante.gouv.fr/fhir/ValueSet/JDV-J66-TypeCode-DMP à la valueset des types de documents possibles

Commit:
#12 #18 Correction sur l'extension AMC afin d'être multi domaine + utilisation du FR Core 2.2.0
